x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/retention/ExpiredForecastsRemover.java

```diff
@@ -100,18 +100,18 @@ public void remove(float requestsPerSec, ActionListener<Boolean> listener, Suppl
     }
 
     private void deleteForecasts(
-        SearchResponse searchResponse,
-        float requestsPerSec,
-        ActionListener<Boolean> listener,
-        Supplier<Boolean> isTimedOutSupplier
+            SearchResponse searchResponse,
+            float requestsPerSec,
+            ActionListener<Boolean> listener,
+            BooleanSupplier isTimedOutSupplier
```

Comment:
I think this indentation change does not match our conventions, can you remove the extra indentation?